.devops/code-review-pipelines.yml

```
@@ -90,16 +90,21 @@ stages:
     jobs:
       - job: integrations_tests
         steps:
-          - template: azure-templates/setup-project.yml
-
-          - script: |
-              yarn generate:definitions
-            displayName: 'Generate code'
+          - template: azure-templates/setup-project.yml'
```

```suggestion
          - template: azure-templates/setup-project.yml
```
Tests are failing because of this, I suppose

---

thanks, done in https://github.com/pagopa/io-functions-commons/pull/159/commits/251bc8f514c09e7bdacd90b1bb37fe94f8fc710d